micov.yml

```diff
@@ -0,0 +1,17 @@
+name: micov
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - click
+  - duckdb
+  - matplotlib>=3.9.0
+  - numba
+  - polars=1.21.0
+  - pyarrow<16.0.0
+  - scipy
+  - tqdm
+  - versioneer
+  - pip:
+      - pip
+      - polars-u64-idx==1.21.0
```

Comment:
```suggestion
```

...or is `polars-u64-idx` not in `conda-forge`?

`polars` and `polars-u64-idx` have a conflicting namespace, so only one can be installed